### ISSUE TITLE
Visited Sites List Remains Stored in Domains Settings

### ISSUE THREAD

**Issue by G-i-o** [Slion/Fulguris#580]
# Description
The option to clear the history of visited sites on exit or manually is defeated by the history being stored under SETTINGS → DOMAINS → ADVANCED.
This must be wiped manually each time.

This is linked to #159.

# Environment
 - OS: Lineage v10
 - Version: 1.9.17

Appreciated! 🙏

**Comment by YOEL-44** [comment]
Yeah, I think there should be some exceptions, you don't want to set everything up every time you open your browser.
But if nothing is changed compared to the default settings, then of course, that webpage should not be listed here after cleaning the browser history.

**Comment by Slion** [comment]
This is mostly fixed since #718.
Simply visiting a site usually does not create a domain settings entry unless there is an SSL security error and user choose to remember his action see: #733.

**Comment by G-i-o** [comment]
Thank You.

Will be looking forward to test the fixed version when out (on F-Droid).

Much appreciated.

**Comment by Slion** [comment]
It should come out on F-Droid soon, its been almost a week.
Once you get it, go to Settings >  Domains and it should list only the domains for which you have actual settings overrides.
The first time you do that it will delete all the useless domain settings file you may have had.
